test/system/index.js

```diff
@@ -1,10 +1,32 @@
 const trader = require('./trader');
-const config = require('./config.json');
-const traders = config.traders;
+const Commander = require('./commander');
 const initTraders = require('./initTraders');
-const scenario1 = require('./runScenario1');
-const scenario2 = require('./runScenario2');
-const scenario3 = require('./runScenario3');
+const TestSingleTrader = require('./scenarios/testSingleTrade');
+const TestTwoTrader = require('./scenarios/testTwoTrader');
+const TestBalances = require('./scenarios/testBalances');
+const DevConfig = require('./configs/config.json');
```

Comment:
this line will fail if ./configs/config.json doesn't exist (see comment in commander.js)